EdFi.Tools.ApiPublisher.Configuration.PostgreSql/PostgreSqlConfigurationValuesProvider.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using Npgsql;
+
+namespace EdFi.Tools.ApiPublisher.Configuration.PostgreSql
+{
+    public class PostgreSqlConfigurationValuesProvider
+    {
+        public IDictionary<string, string> GetConfigurationValues(
+            string connectionString, 
+            string encryptionPassword, 
+            string configurationKeyPrefix)
+        {
+            using var conn = new NpgsqlConnection(connectionString);
```

Comment:
First time I've seen the new pattern-based `using` in the wild. Threw me off for a moment, but I like it. (FYI, I'm not reading most of the code, just scanning through and this caught my eye).